docs/framework/wpf/getting-started/walkthrough-my-first-wpf-desktop-application.md

```diff
@@ -67,11 +67,11 @@ The first step is to create the application infrastructure, which includes an ap
 
     This XAML file defines a WPF application and any application resources. You also use this file to specify the UI that automatically shows when the application starts; in this case, *MainWindow.xaml*.
 
-    Your XAML should look like this in Visual Basic:
+    Example XAML in Visual Basic:
```

Comment:
I as well prefer the original sentence -- I changed it per Acrolinx, and have changed it back.

The tab syntax apparently doesn't allow for included snippet files, only fenced code blocks, so this content is rendered as empty space under the "Visual Basic" heading:

# [Visual Basic](#tab/vb)

    [!code-xaml[ExpenseIt#1_A](~/samples/snippets/visualbasic/VS_Snippets_Wpf/ExpenseIt/VB/ExpenseIt1_A/Application.xaml#1_a)]
---

Changing the language tags from code-xaml to code-vb or code-csharp also causes the code content (snippet) to be rendered blank.

It looks like the original approach is the only feasible one.

Reply:
In my reply above, Github is rendering my example rather than displaying it as entered.  The tab content is actually the following, where the snippet doesn't render:

```
# [Visual Basic](#tab/vb)

    [!code-xaml[ExpenseIt#1_A](~/samples/snippets/visualbasic/VS_Snippets_Wpf/ExpenseIt/VB/ExpenseIt1_A/Application.xaml#1_a)]
---
```